grakn-graql/src/main/java/ai/grakn/graql/internal/query/InsertQueryExecutor.java

```diff
@@ -94,7 +95,7 @@
         varsById = ImmutableMap.copyOf(
                 vars.stream()
                         .filter(var -> var.getId().isPresent())
-                        .collect(Collectors.groupingBy(var -> var.getId().get()))
+                        .collect(Collectors.groupingBy(var -> var.getId().get().getValue()))
```

Comment:
We can definitely make `varsById` be of type `ImmutableMap<ConceptId, List<VarAdmin>>` now.